UhooiPicBook/SceneDelegate.swift

```diff
@@ -85,9 +85,7 @@ extension SceneDelegate {
                 return
         }
 
-        if nav.viewControllers.last is MonsterDetailViewController {
-            nav.viewControllers.removeLast()
-        }
+        nav.popToRootViewController(animated: true)
```

Comment:
```suggestion
        nav.dismiss(animated: false, completion: nil)
        nav.popToRootViewController(animated: true)
```
こんな感じでやればUINavigationController上にpresentされたViewControllerは消せます👐

Reply:
完璧でした、ありがとうございます🙏

949c182
個人的に `completion: nil` は省略したいので消しました！
（↑のコミットは他にも混ぜちゃって見づらいです…）